holoviews/plotting/mpl/element.py

```diff
@@ -149,8 +151,11 @@ def _finalize_axis(self, key, title=None, dimensions=None, ranges=None, xticks=N
         subplots = list(self.subplots.values()) if self.subplots else []
         if self.zorder == 0 and key is not None:
             if self.bgcolor:
-                axis.set_axis_bgcolor(self.bgcolor)
-
+                if LooseVersion(mpl.__version__) <= '1.5.9':
+                    axis.set_axis_bgcolor(self.bgcolor)
```

Comment:
To avoid relying on ``LooseVersion``, maybe we could check that ``2`` is the major version instead? i.e something like ``if mpl.__version__[0] == '2':``? That would also make it clear that the change is occurring in the matplotlib 2+.